src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let cost = 0;
+
+  if (days > 6) {
+    cost = days * 40 - 50;
+  } else if (days > 2 && days < 7) {
+    cost = days * 40 - 20;
+  } else {
+    cost = days * 40;
```

Comment:
Move numbers to variables to make code more readable. For example, 40 => priceForDay, 50 => sevenDaysDiscount etc.